dashboard/modules/job/cli.py

```diff
@@ -24,6 +25,10 @@ def _get_sdk_client(
 ) -> JobSubmissionClient:
     client = JobSubmissionClient(address, create_cluster_if_needed, verify=verify)
     client_address = client.get_address()
+    # redact any passwords in the client address
+    secret = re.findall("https?:\/\/.*:(.*)@.*$", client_address)
```

Comment:
Could you extract the password-redaction functionality into a separate helper function and put the function in [utils.py](https://github.com/ray-project/ray/blob/4c39d9a8f7ece5cca00daaab17077314019c3756/dashboard/modules/job/utils.py)? Then, could you add a unit tests for redacting the password (e.g. redacting a password from a URL with no password, from a URL with a password, from an HTTP URL, and from an HTTPS URL) to [`test_utils.py`](https://github.com/ray-project/ray/blob/4c39d9a8f7ece5cca00daaab17077314019c3756/dashboard/modules/job/tests/test_utils.py)?

Reply:
Thanks for the suggestions! The code has been moved into utils and unit tests have been added in the latest push.